### ISSUE TITLE
Add a command line arguments to set username and password to login at start

### ISSUE THREAD

**Issue by GEEKiDoS** [Ochii/cso2-launcher#18]
<s>To skip the login screen like tiancity server</s>
I managed to get it working with a ugly way:

```cpp
struct LoginParmPointer
{
	uint32_t padding[0x4]; // 0-0Fh
	char** userName; // 10h-13h
	uint32_t padding1[0x7]; // 14h-29h
	char** passWord; // 30h-33h
};

struct LoginParm
{

	uint32_t padding[0x2]; // 0-7
	LoginParmPointer *values; // 8-0Bh
	uint32_t valueSize; // 0Ch-0Fh
};

// Run those code in CSO2UIManager_InitMainUI(client.dll + 0xAE4610)
	auto newvalues = new LoginParm;
	newvalues->values = new LoginParmPointer;
	newvalues->valueSize = 2u;
	char *username = "123456789";
	//max 4
	char *password = "1234";
	newvalues->values->userName = &username;
	newvalues->values->passWord = &password;

	((void(__cdecl*)(LoginParm *values))client.dll + 0xACA490)(newvalues);
```
Then the game will login as 123456789 with password 1234 at start.

@Ochii

**Comment by organnerx** [comment]
You can skip login screen by pressing home button at the top right. Then it will be bugged and you will not have a nickname, character and test server in the list. Looks like a bug

**Comment by GEEKiDoS** [comment]
I managed to get it working with a ugly way:

```cpp
struct LoginParmPointer
{
	uint32_t padding[0x4]; // 0-0Fh
	char** userName; // 10h-13h
	uint32_t padding1[0x7]; // 14h-29h
	char** passWord; // 30h-33h
};

struct LoginParm
{

	uint32_t padding[0x2]; // 0-7
	LoginParmPointer *values; // 8-0Bh
	uint32_t valueSize; // 0Ch-0Fh
};

// Run those code in CSO2UIManager_InitMainUI(client.dll + 0xAE4610)
	auto newvalues = new LoginParm;
	newvalues->values = new LoginParmPointer;
	newvalues->valueSize = 2u;
	char *username = "123456789";
	//max 4
	char *password = "1234";
	newvalues->values->userName = &username;
	newvalues->values->passWord = &password;

	((void(__cdecl*)(LoginParm *values))client.dll + 0xACA490)(newvalues);
```
Then the game will login as 123456789 with password 1234 at start.

**Comment by Ochii** [comment]
Merged in [`08473fd`](/Ochii/cso2-launcher/commit/08473fd850a0e8380211ce7cd0abacd67119d348)
